packages/api/src/controllers/user.ts

```diff
@@ -449,6 +449,109 @@ app.post("/", validatePost("user"), async (req, res) => {
   res.json(user);
 });
 
+app.patch("/", authorizer({}), async (req, res) => {
+  const { email } = req.body;
+  const userId = req.user.id;
+
+  // Validate the new email
+  const emailValid = validator.validate(email);
+  if (!emailValid) {
+    return res.status(422).json({ errors: ["Invalid email"] });
+  }
```

Comment:
Make sure to lowercase the email as well
`const lowercaseEmail = email.toLowerCase();`